doc/man3/OSSL_PARAM_allocate_from_text.pod

```diff
@@ -26,34 +26,39 @@ OpenSSL 3.0 introduces a new mechanism to do the same thing with an
 array of parameters that contain name, value, value type and value
 size (see L<OSSL_PARAM(3)> for more information).
 
-OSSL_PARAM_allocate_from_text() takes a control I<key>, I<value> and
-value size I<value_n>, and given a parameter descriptor array
-I<paramdefs>, it converts the value to something suitable for
-L<OSSL_PARAM(3)> and stores that in the buffer I<buf>, and modifies
-the parameter I<to> to match.
-I<buf_n>, if not NULL, will be assigned the number of bytes used in
-I<buf>.
-If I<buf> is NULL, only I<buf_n> will be modified, everything else is
-left untouched, allowing a caller to find out how large the buffer
-should be.
-I<buf> needs to be correctly aligned for the type of the B<OSSL_PARAM>
-I<key>.
-If <found> is not NULL, it is set to 1 if the parameter can be located and
-to 0 otherwise.
-
-The caller must remember to free the data of I<to> when it's not
-useful any more.
+OSSL_PARAM_allocate_from_text() looks up I<key> in I<paramdefs>.
+If an item was found, it converts I<value> (I<value_n> bytes) to
+something suitable for that item's I<data_type>, and stores the
+result in I<< to->data >> as well as its size in I<< to->data_size >>.
+I<< to->key >> and I<< to->data_type >> are copied from the item that
+was found, and I<< to->return_size >> is set to zero.
```

Comment:
If to->key is copied, what does that mean for ownership of the pointer?

Reply:
Later you document that if you pass "hexkey", it will contain "key", so I assume it's actually allocated, and needs to be freed too?

Reply:
Uhm, no. The pointer *to-\>key* becomes the same as the *key* pointer from the item that was found.

Reply:
Does it help if I say "assigned" instead of "copied"?  See the text change I made